tests/test_admin.py

```diff
@@ -141,6 +141,46 @@ def test_template_filter(self):
         self.assertSetEqual(set(response_template_1.context["cl"].queryset), set(template_1_pages))
         self.assertSetEqual(set(response_template_2.context["cl"].queryset), set(template_2_pages))
 
+    def test_author_filter(self):
+        """
+        Author filter should only show selected author's results
+        """
+        author1 = UserFactory()
+        author2 = UserFactory()
+        page_author_1 = PageVersionFactory(content__template="page.html", content__language="en", created_by=author1)
+        page_author_2 = PageVersionFactory(content__template="page.html", content__language="en", created_by=author2)
+
+        author_selection = f"?created_by={author1.pk}"
```

Comment:
Could you change this variable name to something like author_param?

Reply:
Done